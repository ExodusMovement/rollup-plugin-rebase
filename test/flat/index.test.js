/* global __dirname */
import { bundle, clean, exists, list } from "../util"

const root = __dirname

test("Flat", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(list(root, "output")).resolves.toMatchSnapshot()

  expect(exists(root,
    [
      "output/index.js",
      "output/bsdudxaF.md"
    ])).resolves.toBeTruthy()

  await clean(root, "output")
})
