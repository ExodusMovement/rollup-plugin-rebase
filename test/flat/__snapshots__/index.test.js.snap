// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flat 1`] = `
"bsdudxaF.md
index.js"
`;
